### ISSUE TITLE
I would strongly recommend that Version of library would be mandatory information

### ISSUE THREAD

**Issue by mWaltari** [Memnarch/Delphinus#35]
Now I have no idea which version of Spring4D it would install. (Or Delphinus does not show the version info for the library).

**Comment by Memnarch** [comment]
UI or commandline? UI should show it in the combobox like here:
http://memnarch.bplaced.net/blog/2016/01/delphinus-new-ui-and-no-external-ssl-libraries-anymore/
and the commandline prints the version in one of the first lines.

**Comment by mWaltari** [comment]
Yes, it was "hidden" under the installation button, which I did not dare to click :)

Maybe adding some kind ov available version list or popup menu etc, to the panel on the right...

**Comment by mWaltari** [comment]
Just pondering would there be some subtle hint (Way the GUI would look or so) that Install button would tell user that it is not going to start right away. As I thought it would be.

I'll tell if I get and ultimate idea for this :)

**Comment by Memnarch** [comment]
AH ok.
I'll look into it. The problem is(while it is not yet exactly that way), the idea is not to load the version-history for every package. Because besides loading the version, i have to look into the Delphinus.info of that version, to determine if this version is still available for your Delphi-Version. For example if you use XE and the most recent version does not Support XE anymore. It is a lot of additional calls to the GithubApi. But you're right, i should clarify this somehow in the UI that it won't just install right away.
